topics/single-cell/tutorials/scrna-umis/tutorial.md

```diff
@@ -44,7 +44,7 @@ recordings:
 Barcodes are small oligonucleotides that are inserted into the captured sequence at a specific point, and provide two pieces of information about the sequence:
 
  1. Which cell the sequence came from
- 2. Which transcript the sequence came from
+ 2. Which transcript the sequence came from?
```

Comment:
reading the sentence before it, maybe it should just be a period at the end of both items, what do you think?

Reply:
or, if you want it to be a question, rephrase it "Which trascript **did** the sequence **come** from?"